Cycles/config.py

```diff
@@ -24,6 +24,7 @@
 			" -D WITH_CYCLES_PATH_GUIDING=ON"
 			" -D WITH_CYCLES_CUDA_BINARIES=ON"
 			" -D WITH_CYCLES_DEVICE_CUDA=ON"
+			" -D CYCLES_CUDA_BINARIES_ARCH='sm_60;sm_61;sm_70;sm_75;sm_86;sm_89;compute_75'"
```

Comment:
Does this put us on the hook for noticing when new architectures become available? If we missed a new architecture, would Cycles fail to run on it, or would it silently be falling back to an older architecture that might be less optimal?

Reply:
According to [this](https://archive.blender.org/developer/D8332), because we're still building the `compute_75` PTX binary, any architecture newer than Turing (RTX 20x0) should be able to JIT-compile the PTX, so newer architectures without a specific `sm_xx` binary should still work with the caveat of a slower startup time.

As discussed offline, I've switched this over to a patch of `CMakeLists.txt` so it's more obvious to us when newer architectures are added.